### PR TITLE
Fix links to C# troubleshooting guide

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -41,14 +41,13 @@ If you would like to actually debug through a language generator's code, see our
 [main_docs]: https://github.com/Azure/autorest/blob/master/docs/openapi/readme.md
 [autorest_issues]: https://github.com/Azure/autorest/issues
 [autorest_python_issues]: https://github.com/Azure/autorest.python/issues
-[main_debugging]: https://github.com/Azure/autorest/tree/master/docs/generate/troubleshooting.md#debugging
```
Comment (timotheeguerin) [inline docs/troubleshooting.md:44]: I think this one needs to be updated to https://github.com/Azure/autorest/blob/master/docs/troubleshooting.md#debugging

Comment (kant2002) [reply docs/troubleshooting.md:44]: I doube-check that this link do not referenced anywhere in any form on this project.

Comment (kant2002) [reply docs/troubleshooting.md:44]: See https://github.com/Azure/autorest/search?q=main_debugging for convinience.

Comment (timotheeguerin) [reply docs/troubleshooting.md:44]: right... this is a reference to itself

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**